docs/tempo/website/api_docs/_index.md

```diff
@@ -91,13 +91,18 @@ Tempo's Query API is simple. The following request is used to retrieve a trace f
 a microservices deployment, or the Tempo endpoint in a monolithic mode deployment.
 
 ```
-GET /api/traces/<traceid>
+GET /api/traces/<traceid>?start=<start>&end=<end>
 ```
+Parameters:
+- `start = (unix epoch seconds)`
+  Optional.  Along with `end` define a time range from which traces should be returned. 
+- `end = (unix epoch seconds)`
+  Optional.  Along with `start` define a time range from which traces should be returned. Providing both `start` and `end` will include traces for the specified time range only. 
```

Comment:
Can we move the extended warning and description below here? This is considered the external endpoint and it's the one we really need that warning on.